app/models/manageiq/providers/container_manager.rb

```diff
@@ -22,6 +22,12 @@ class ContainerManager < BaseManager
     has_one :container_deployment, :foreign_key => :deployed_ems_id, :inverse_of => :deployed_ems
     has_many :computer_systems, :through => :container_nodes
 
+    # Archived entities to destroy when the container manager is deleted
+    has_many :old_container_groups, :foreign_key => :old_ems_id, :dependent => :destroy
+    has_many :old_containers, :foreign_key => :old_ems_id
+    has_many :old_container_definitions, :foreign_key => :old_ems_id
```

Comment:
@moolitayer destroying `container_group` that destroys `container_definition` that destroys `container`